libs/non-integral/non-integral.cabal

```diff
@@ -23,7 +23,7 @@ library
         -Wall -Wcompat -Wincomplete-record-updates
         -Wincomplete-uni-patterns -Wredundant-constraints -Wunused-packages
 
-    build-depends:    base >=4.14 && <4.19
+    build-depends:    base >=4.14 && <5
```

Comment:
Change like this requires a patch version update. Could you please bump up the version of `non-integral` in the cabal file from `1.0.0.0` to `1.0.0.1`